docs/packaging/procedures/migrate-repos-to-github.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Migrating Existing Repos to GitHub
+summary: Migrate Your Existing Repos From Phab to GitHub
+---
+
+# How To Migrate Pre-existing Repos and `common` to GitHub
+
+:::info
+This is intended for people who have cloned package repos from Phabricator in the past. The information below will help you migrate your `common` repo and existing repo clones to work with our GitHub.
+:::
+
+This document assumes your packages are stored under the folder `~/solus-builds`, adjust the commands to match the directory name on your system.
+
+People who have set up repos before the migration to GitHub will have git remote URLs pointing to dev.getsol.us. To see if this is the case, change to the common directory, and run the following to double check:
+
+``` bash
+cd ~/solus-builds/common
+❯ git remote -v
+origin  https://dev.getsol.us/source/common.git (fetch)
+origin  https://dev.getsol.us/source/common.git (push)
+
+```
+
+You will need to make sure you are on the master branch and update the git remote URLs:
+
+```bash
+cd ~/solus-builds/common
+git switch master
+git remote set-url origin https://github.com/getsolus/common
+git remote set-url --push origin git@github.com:getsolus/common.git
```

Comment:
https URL for one and ssh URL for the other? Why not both be the same?

Reply:
This is what I was given in the Solus Matrix channels. I don't know why its this way. If Silke agrees it should be changed, it can go into his docs update for the monorepo.

Reply:
We can revisit this later. Let's not block on it.